### ISSUE TITLE
Cannot find the added file when reopening terminal

### ISSUE THREAD

**Issue by achxyz** [jdunic/Intro-To-Practical-Computing-R#5]
Hi, I have a problem that I cannot find the added file when I reopen the terminal. Before I close it, it went well as Jillian showed in class. But when I changed the preference of terminal, and then reopen the terminal, the added file I kept track of before cannot be found. Here is several lines of what I did when I reopen terminal:
![screenshot 2015-09-10 10 17 10](https://cloud.githubusercontent.com/assets/7248636/9790640/22510d0a-57a5-11e5-9a2f-6d6599e8f265.jpg)

Did anyone have the same problem as I had? Thank you if anyone could help to solve this!

**Comment by bseitz234** [comment]
Do you see the file if you `ls` in your homework folder? I'm pretty sure git can only add an untracked file, and since it didn't come up on the untracked files the first time, it probably wasn't able to add it. 

It may also be that you haven't made any changes since your last commit- so it's not really in the staging area or untracked.

**Comment by achxyz** [comment]
Yeah, I think you're right. But I cannot see the added one in status, which is wired, I can see it before. When I did ls, three files showed including My_first_story.rmd, when I tried to make another comment (I made two before I first closed terminal), but there was no note for the third comment in git log... 
![screenshot 2015-09-10 10 27 39](https://cloud.githubusercontent.com/assets/7248636/9790962/d2d4226a-57a6-11e5-979e-22f7ce14f025.jpg)

**Comment by bseitz234** [comment]
Did you change My_first_story.rmd before trying that third commit? If the file was unchanged, then there would have been nothing to update. Try adding another line to the file in RStudio, and then you should see it as a modified file when you run `git status` and be able to add it and commit that new change.

**Comment by achxyz** [comment]
Yes, it showed up! I made some modifications in my My_first_story file in RStudio, then when I typed git status in terminal, the added file appeared. Thanks a lot!
